src/util/index.ts

```diff
@@ -1,25 +1,37 @@
 /* eslint-env browser */
 
-module.exports.baseUrl =
+declare global {
```

Comment:
We might want to create our own module with `.d.ts` files in the future, so we can centralize some of our types that may be shared across other modules.

Reply:
I'm going to punt this down the road a bit - agree that centralized types will be good (strongly-typed APIs? yes pls!) but we can make that happen as it's needed.